source/geod24/LocalRest.d

```diff
@@ -820,6 +823,24 @@ public final class RemoteAPI (API, alias S = VibeJSONSerializer!()) : API
         {
             C.send(this.childTid, FilterAPI(""));
         }
+
+        /***********************************************************************
+
+            Call the provided delegate with a scoped copy of this RemoteAPI
+            with a custom timeout.
+
+            Params:
+                timeout = the new timeout to use
+                dg = the delegate to call with the new scoped RemoteAPI copy
+
+        ***********************************************************************/
+
+        public void withTimeout (Dg) (Duration timeout, Dg dg) @trusted
```

Comment:
```suggestion
        public void withTimeout (Dg) (Duration timeout, scope Dg dg)
```

Reply:
By the way the signature I had originally in mind was `withTimeout(alias Callable)(Duration timeout)` but this works too

Reply:
It's unfortunate ldc2 errors with the "need dual context support", otherwise yes it would look really nice that way.